doc-surrealdb_versioned_docs/version-latest/tutorials/integrate-auth0-as-authentication-provider.mdx

```diff
@@ -139,6 +139,9 @@ USE NS test DB test;
 DEFINE ACCESS auth0 ON DATABASE TYPE RECORD
   -- We check the token claims and map the email address to a record user.
   AUTHENTICATE {
+    -- The JWT specification allows the audience claim to be an array or a string.
+    -- In this example, we ensure that it is provided as an array by Auth0.
+    type::is::array($token.aud) AND
```

Comment:
Added this change here: https://github.com/surrealdb/docs.surrealdb.com/pull/760/files#diff-e2536a3000bd4fc687e0b912f36a4f93cabcc999f7dc2abc8a9068d0bbf76c4bR252

Reply:
Makes sense. What about the rest of the changes? Should I remove this one to merge the rest?

Reply:
Oh no just this one. I can update and merge it.